.travis_scripts/build_all

```diff
@@ -27,6 +27,7 @@ echo "Configuring conda."
 conda config --add channels conda-forge
 conda config --set show_channel_urls true
 conda install --yes --quiet "conda!=4.6.1" conda-forge-ci-setup=2.* conda-forge-pinning networkx conda-build>=3.16
+
```

Comment:
These modifications outside the recipe need to be removed.

Reply:
ready